src/SpineOpt.jl

```diff
@@ -128,7 +129,13 @@ include("constraints/constraint_node_state_capacity.jl")
 include("constraints/constraint_ratio_unit_flow.jl")
 include("constraints/constraint_ratio_out_in_connection_flow.jl")
 include("constraints/constraint_connection_flow_capacity.jl")
-include("constraints/constraint_connection_flow_ptdf.jl")
+include("constraints/constraint_connection_flow_capacity_strict.jl")
+include("constraints/constraint_connection_intact_flow_capacity.jl")
+include("constraints/constraint_connection_flow_intact_flow.jl")
+include("constraints/constraint_connection_intact_flow_ptdf.jl")
+include("constraints/constraint_connection_intact_flow_ptdf_in_out.jl")
+include("constraints/constraint_candidate_connection_flow_intact_flow.jl")
+include("constraints/constraint_candidate_connection_flow_intact_flow2.jl")
```

Comment:
Would you consider:

constraint_candidate_connection_flow_intact_flow -> constraint_candidate_connection_flow_lower_bound

constraint_candidate_connection_flow_intact_flow2 -> constraint_candidate_connection_flow_upper_bound

Reply:
that's nicer all right